### PR TITLE
Trigger `auslese:change` event on original, automounted `<select>`

### PR DESCRIPTION
**Pull request by keichinger**
| Q             | A                                                                     |
| ------------- | --------------------------------------------------------------------- |
| BC breaks?    |no                                                                |
| New feature?  | yes                   |
| Improvement?  |no     |
| Bug fix?      |no                                                                |
| Deprecations? |no     |
| Docs PR       | **missing**                                   |

### CODE REVIEW

File: js/automount.ts [hunk 1]
```diff
@@ -198,6 +200,10 @@ function updateSelectState (
     options.forEach(option => {
         option.selected = selection[option.value];
     });
+
+    trigger(select, "auslese:change", {
```
Comment (apfelbox) [inline js/automount.ts]: You should export the event name as constant

**Review by apfelbox — CHANGES_REQUESTED** [review]

**Comment by keichinger** [comment]
Honestly, I have no idea how to fix the cypress error. I've tried the suggestions that I found in various issues over on the cypress org. Apparently, they don't seem to work for us 🤷‍♂️

**Review by apfelbox — APPROVED** [review]

**Comment by apfelbox** [comment]
The issue was that you didn't update the cache key. So while your changes were correct, it still always used the old (immutable) cache.

So one fix is to just bump the cache. A fail-safe is manually calling `cypress install` in the CI, which *should* be a no-op if it was already installed by `npm install`.

**Comment by apfelbox** [comment]
2020-07-28 at 15 45

It **is** a no-op

**Pull request merged**